app/controllers/offers_controller.rb

```diff
@@ -59,7 +62,8 @@ def batch_email_nags
         if offer
           offer.increment!(:nag_count, 1)
           if ENV['RAILS_ENV'] != 'test'
-            CpMailer.nag_email(offer.format).deliver_now
+            mangled = offer[:link]
```

Comment:
at the head of this function should say that it is the action that admins trigger when they want to send nag emails to a selected set of contracts that have been presented but not accepted or rejected.. (but here in the back end we don't check that they are still not accepted, etc